src/homework1.py

```diff
@@ -0,0 +1,131 @@
+import itertools
+import random
+import re
+
+# 1 line: Output
```

Comment:
It looks much more clear if you task description stands closely to task and 
separated from other tasks with two blank lines: 
```python
   pass  # last line of previous task
    
 
   # next task docstring
   pass  # my next task first line  
```